Header header
string uav_name
float32 orbit_angle
